no_cluster.robot

```diff
@@ -0,0 +1,79 @@
+*** Settings ***
```

Comment:
Can we call this file just repos.robot? There might be more repo stuff to test, and also alot more non-cluster stuff that can pushed into other files